### PR TITLE
Release562

### PR DESCRIPTION
**Pull request by jburel**
cc @mtbc @sbesson 
prep for coming release

### CODE REVIEW

File: _posts/2020-07-16-omero-5-6-2.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: Release of OMERO.server 5.6.2 and OMERO.web 5.7.0
+intro-blurb: This release includes OMERO.server 5.6.2, OMERO.web 5.7.0 and OMERO.py 5.7.1
+---
+
+Today we are releasing OMERO.server 5.6.2 which fixes installaion instructions for CentOS 7,
```
Comment (sbesson) [inline _posts/2020-07-16-omero-5-6-2.md:7]:
```suggestion
Today we are releasing OMERO.server 5.6.2 which fixes installation instructions for CentOS 7, 
```

Also maybe "the documentation fixes the installation instructions" or were changes also made in the server bundle itself?

File: _posts/2020-07-16-omero-5-6-2.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+We are also releasing OMERO.web 5.7.0 and OMERO.py 5.7.1 which both contain some API additions.
+
+OMERO.server 5.6.2 runs in the same deployment environments as OMERO.server 5.6.0 and
+the Bio-Formats Memoizer cache will be invalidated.
+
+### Installing the Software:
+
+For full details of the changes with the OMERO 5.6 series see the
+[OMERO 5.6.0]({{ site.baseurl }}/2020/01/15/omero-5-6-0.html) release
+announcement. OMERO.server 5.6.1 is designed to be a small step beyond the 5.6.0 release
```
Comment (sbesson) [inline _posts/2020-07-16-omero-5-6-2.md:19]:
```suggestion
announcement. OMERO.server 5.6.2 is designed to be a small step beyond the 5.6.0 release
```

File: _posts/2020-07-16-omero-5-6-2.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+in order to simplify upgrade. Full documentation for this release is available
+under <https://docs.openmicroscopy.org/omero/5.6.1/>.
```
Comment (sbesson) [inline _posts/2020-07-16-omero-5-6-2.md:21]:
```suggestion
under <https://docs.openmicroscopy.org/omero/5.6.2/>.
```

File: _posts/2020-07-16-omero-5-6-2.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+OMERO.server 5.6.2 is available from
+[archived downloads](https://downloads.openmicroscopy.org/omero/5.6.2/artifacts/)
+and [omero-web 5.7.0](https://pypi.org/project/omero-web/5.7.0/).
+These have been tested with
+[omero-py 5.7.1](https://pypi.org/project/omero-py/5.7.1/) so we
+recommend that you upgrade OMERO.py on both server and web deployments.
```
Comment (sbesson) [inline _posts/2020-07-16-omero-5-6-2.md:28]: For OMERO.web, is `omero-py` going to be automatically upgraded?

File: _posts/2020-07-16-omero-5-6-2.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: Release of OMERO.server 5.6.2 and OMERO.web 5.7.0
```
Comment (sbesson) [inline _posts/2020-07-16-omero-5-6-2.md:3]: :+1: One could imaging including OMERO.iviewer as the latest release consume API additions from OMERO.web 5.7.0. Has this been discussed?

File: _posts/2020-07-16-omero-5-6-2.md (hunk 4 above)
Comment (jburel) [reply _posts/2020-07-16-omero-5-6-2.md:28]: only if you run ``--upgrade``

File: _posts/2020-07-16-omero-5-6-2.md (hunk 5 above)
Comment (jburel) [reply _posts/2020-07-16-omero-5-6-2.md:3]: yes, I discussed with @will-moore and he has prepared a PR https://github.com/ome/omero-iviewer/pull/331

File: _posts/2020-07-16-omero-5-6-2.md (hunk 4 above)
Comment (sbesson) [reply _posts/2020-07-16-omero-5-6-2.md:28]: Even with https://github.com/ome/omero-web/commit/c2c419865f0afaf84f84c712d2b46c11ccd8c3d4 ?

Comment (jburel) [reply _posts/2020-07-16-omero-5-6-2.md:28]: it should be updated. i forgot about that condition

File: _posts/2020-07-16-omero-5-6-2.md (hunk 5 above)
Comment (jburel) [reply _posts/2020-07-16-omero-5-6-2.md:3]: Do we want to combine all the release in one announcement?

**Comment by jburel** [comment]
for peace of mind 
I installed ``omero-py < 5.7.0`` in a conda env then installed ``omero-web``. ``omero-py`` was upgraded
```
Attempting uninstall: omero-py
    Found existing installation: omero-py 5.6.2
    Uninstalling omero-py-5.6.2:
      Successfully uninstalled omero-py-5.6.2
Successfully installed Django-1.11.29 django-pipeline-1.6.14 gunicorn-20.0.4 omero-marshal-0.7.0 omero-py-5.7.1 omero-web-5.7.0 pytz-2020.1
```

File: _posts/2020-07-16-omero-5-6-2.md (hunk 5 above)
Comment (sbesson) [reply _posts/2020-07-16-omero-5-6-2.md:3]: Given that OMERO.iviewer 0.10.0 requires OMERO.web 5.7.0 I think it makes sense to combine them.

File: _posts/2020-07-16-omero-5-6-2.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+We are also releasing OMERO.web 5.7.0 and OMERO.py 5.7.1 which both contain some API additions.
+
+OMERO.server 5.6.2 runs in the same deployment environments as OMERO.server 5.6.0 and
+the Bio-Formats Memoizer cache will be invalidated.
+
+### Installing the Software:
+
+For full details of the changes with the OMERO 5.6 series see the
+[OMERO 5.6.0]({{ site.baseurl }}/2020/01/15/omero-5-6-0.html) release
+announcement. OMERO.server 5.6.2 is designed to be a small step beyond the 5.6.0 release
```
Comment (mtbc) [inline _posts/2020-07-16-omero-5-6-2.md:19]: I think this sentence is now redundant but I don't mind it. The point was to not inhibit people from adopting the security release.

File: _posts/2020-07-16-omero-5-6-2.md [hunk 7]
```diff
@@ -16,9 +16,7 @@ the Bio-Formats Memoizer cache will be invalidated.
 
 For full details of the changes with the OMERO 5.6 series see the
 [OMERO 5.6.0]({{ site.baseurl }}/2020/01/15/omero-5-6-0.html) release
-announcement. OMERO.server 5.6.2 is designed to be a small step beyond the 5.6.0 release
-in order to simplify upgrade. Full documentation for this release is available
-under <https://docs.openmicroscopy.org/omero/5.6.2/>.
+announcement.
```
Comment (mtbc) [inline _posts/2020-07-16-omero-5-6-2.md:19]: Do we still want to mention where to find full documentation though?

File: _posts/2020-07-16-omero-5-6-2.md [hunk 8]
```diff
@@ -19,8 +19,8 @@ For full details of the changes with the OMERO 5.6 series see the
 announcement. 
 
 OMERO.server 5.6.2 is available from
-[archived downloads](https://downloads.openmicroscopy.org/omero/5.6.2/artifacts/)
-and [omero-web 5.7.0](https://pypi.org/project/omero-web/5.7.0/).
+[archived downloads](https://downloads.openmicroscopy.org/omero/5.6.2/artifacts/).
+[omero-web 5.7.0](https://pypi.org/project/omero-web/5.7.0/) and OMERO.iviewer 0.10.0](https://pypi.org/project/omero-iviewer/0.10.0/) are both available from PyPI.
```
Comment (sbesson) [inline _posts/2020-07-16-omero-5-6-2.md]:
```suggestion
[OMERO.web 5.7.0](https://pypi.org/project/omero-web/5.7.0/) and [OMERO.iviewer 0.10.0](https://pypi.org/project/omero-iviewer/0.10.0/) are both available from PyPI.
```

File: _posts/2020-07-16-omero-5-6-2.md [hunk 9]
```diff
@@ -19,8 +19,8 @@ For full details of the changes with the OMERO 5.6 series see the
...
 These have been tested with
 [omero-py 5.7.1](https://pypi.org/project/omero-py/5.7.1/) so we
```
Comment (sbesson) [inline _posts/2020-07-16-omero-5-6-2.md:25]:
```suggestion
[OMERO.py 5.7.1](https://pypi.org/project/omero-py/5.7.1/) so we
```

**Pull request merged**